app/main/posts/modify/post-tabs.directive.js

```diff
@@ -4,16 +4,15 @@ PostVerticalTabs.$inject = [];
 function PostVerticalTabs() {
     return {
         restrict: 'E',
-        replace: true,
```

Comment:
The directive was failing to load scope for the page, the replace: true was the cause. The reason the completion toggle was not working was because it was not calling through to the function.